### PR TITLE
TYP: add mypy messages as comments to ignores

### PR DESCRIPTION
**Pull request by simonjayhawkins**
xref #37715

### CODE REVIEW

File: pandas/core/arrays/floating.py [hunk 1]
```diff
@@ -78,7 +78,10 @@ def _get_common_dtype(self, dtypes: List[DtypeObj]) -> Optional[DtypeObj]:
         if not all(isinstance(t, FloatingDtype) for t in dtypes):
             return None
         np_dtype = np.find_common_type(
-            [t.numpy_dtype for t in dtypes], []  # type: ignore[union-attr]
+            # error: Item "ExtensionDtype" of "Union[Any, ExtensionDtype]" has no
```
Comment (jbrockmendel) [inline pandas/core/arrays/floating.py:81]: could do a cast here since we have all FloatingDtype

Comment (simonjayhawkins) [reply pandas/core/arrays/floating.py:81]: just wanted to get the messages up-to-date in this PR to help with https://github.com/MarcoGorelli/pyladies-sprint-March-2021/issues/5 (and subsequent reviews). any fixes to be done separately

Comment (jbrockmendel) [reply pandas/core/arrays/floating.py:81]: works for me

**Comment by jbrockmendel** [comment]
thanks @simonjayhawkins

**Pull request merged**